py-polars/Makefile

```diff
@@ -47,14 +49,13 @@ develop: $(VENV)  ## Build by running `maturin develop`
 	$(VENV_ACT_BIN)maturin develop
 
 develop-release: $(VENV)  ## Build by running `maturin develop --release`
-	$(VENV_ACT_BIN)maturin develop --release
+	$(VENV_ACT_BIN)maturin develop $(RELEASE_FLAGS)
 
 .PHONY: clean cleaner cleanest
 clean:  ## Just run `cargo clean`
```

Comment:
I'm not fond of the division of `make clean` into 3 stages. If you want to run `cargo clean`, you can just run `cargo clean`. Not everything needs to be a `make` command.

Reply:
Makefile standard practice is to provide several cleaning targets like `clean`, `cleaner`, `cleanest`, `clean-dist`, etc. They serve specific needs that cannot be achieved by a single `clean` target.

That said, I see your point that `make clean` does not add anything over `cargo clean` and I will refactor this portion of the Makefile eliminating such a redundancy.